docs/content/using-tesseracts/advanced.md

```diff
@@ -66,12 +66,12 @@ As an alternative to the MLflow setup we provide, you can point your Tesseract t
 $ tesseract serve --env=TESSERACT_MLFLOW_TRACKING_URI="..."  metrics
 ````
 
-Note that if your MLFlow server uses basic auth, you need to populate the `TESSERACT_MLFLOW_TRACKING_USERNAME` and
-`TESSERACT_MLFLOW_TRACKING_PASSWORD` for the Tesseract to be able to authenticate to it.
+Note that if your MLFlow server uses basic auth, you need to populate the `MLFLOW_TRACKING_USERNAME` and
+`MLFLOW_TRACKING_PASSWORD` for the Tesseract to be able to authenticate to it.
```

Comment:
```suggestion
`MLFLOW_TRACKING_PASSWORD` env variables for the Tesseract to be able to authenticate to it.
```
(sorry, had to do in two comments because for some reason multi-line suggestion on gh is not working 🤷 )